sql/core/src/test/scala/org/apache/spark/sql/collation/CollationTypePrecedenceSuite.scala

```diff
@@ -110,6 +110,54 @@ class CollationTypePrecedenceSuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("lateral alias has implicit strength") {
+    checkAnswer(
+      sql(s"""
```

Comment:
interpolation is not needed here. could you remove s":
```suggestion
      sql("""
```